src/datasets/arrow_dataset.py

```diff
@@ -159,6 +159,51 @@ def version(self):
         return self._info.version
 
 
+class TensorflowDatasetMixIn:
+    def __init__(self):
+        pass
+
+    def to_tf_dataset(self, tokenizer, cols_to_remove, batch_size, shuffle):
+        import tensorflow as tf
+        dataset_in = self.remove_columns(cols_to_remove)
+        tf_cols = [col for col in dataset_in.features]
+        label_index = tf_cols.index("label")
```

Comment:
We should probably have a `label_cols`  argument since it's not always `"label"` and there could be more than one. QA models use `"start_positions"` and `"end_positions"` for instance

Reply:
Done! We support (potentially overlapping) lists of columns and label_cols now